### ISSUE TITLE
PWP instances versus states?

### ISSUE THREAD

**Issue by hlflanagan** [w3c/dpub-pwp-ucr#80]
This might be a question of unclear definitions (PWP states vs PWP instances).
“This does not imply that the same PWP in different states is required to use the same identifier.” 
And 
“Identifiers need to be persistent across PWP instances.”
--http://w3c.github.io/dpub-pwp-ucr/#r_identifier

**Comment by bjdmeest** [comment]
I am no expert, but I assume it is possible that the same PWP (so same contents) in different states can have different identifiers (e.g., different ISBNs for print vs digital) (which is what the note is about)
Maybe rephrase to:
Req. 35: Identifiers must be persistent across instances and maybe across states, but must not conflict with locators.

Identification of a publication is orthogonal to the issue of locators. There are no specific requirements on identification of PWPs in this document in general. Identifiers need to be persistent across PWP instances. Using the same identifier across states should also be possible but not obligatory and must not conflict with locators.

(and then you can remove the note imo)

**Comment by lrosenthol** [comment]
think I cleaned this all up in the updated version
